src/styles/blocks/_watch.scss

```diff
@@ -0,0 +1,87 @@
+.watch {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  margin-top: 50px;
+
+  &__circle {
+    @include circle(600px);
+
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size: 400% 400%;
+
+    animation: blobs 15s ease-in-out infinite alternate,
+      bg-gradient 15s ease infinite;
+    transition: 2s;
```

Comment:
You should set transition properties that are going to change, so the browser doesn't need to watch all properties

Reply:
agree, let's fix it